### PR TITLE
Document use on ANSI with mkdocs

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

File: docs/stories/from-sphinx-to-mkdocs.md [hunk 1]
```diff
@@ -104,6 +104,29 @@ and thrive. One notable mention is that "blog" feature seems to be for insiders
...
+### Rendering ANSI output from external commands
+
+You need the following steps
+
+- Install `pygments-ansi-color` and `markdown-exec` packages
+- Add `markdown-exec` to `plugins` in `mkdocs.yml`
+- Add `extra_css` for stylesheet like
+  [this](https://github.com/42picky/42picky.github.io/blob/main/docs/stylesheets/extra.css)
+  for the colors.
+- Use `ansi` language for code blocks or `return="ansi"` for markdown-exec.
```
Comment (pawamoy) [inline docs/stories/from-sphinx-to-mkdocs.md:116]: I think it's `result` instead of `return`.

File: docs/stories/from-sphinx-to-mkdocs.md [hunk 2]
```diff
@@ -182,13 +205,13 @@ files into markdown code snippets.
 
 While there are at least two solutions for including content with [mkdocs], they
 both are problematic because some decisions that were made regarding how
-relative paths are computed, something that breaks badly when you try to build a
+relative paths are computer, something that breaks badly when you try to build a
```
Comment (pawamoy) [inline docs/stories/from-sphinx-to-mkdocs.md:208]: Heh, automatic spelling fixes? I reviewed them last time, and they went back to the previous, wrong version x)

File: docs/stories/from-sphinx-to-mkdocs.md [hunk 3]
```diff
@@ -182,13 +205,13 @@ files into markdown code snippets.
...
 single site out of multiple repositories. I am even wondering if this issue is
-fixable as it might touch the core concepts on which [mkdocs] is built.
+fixable as it might touch the core concepts on which [mkdocs] is build.
 
 It seems that mkdocs does not provide plugin authors with a way to get the path
 of the current document and instead it provides two paths, the
-`docs_path: docs/` and the configuration path. Not only this but it states
+`docs_path: docs/` and the path configuration path. Not only this but it states
```
Comment (pawamoy) [inline docs/stories/from-sphinx-to-mkdocs.md:214]: Hmm, looks like you committed old contents instead? This is not a spelling mistake but was corrected previously as well.

File: mkdocs.yml [hunk 4]
```diff
@@ -83,3 +84,7 @@ markdown_extensions:
         - name: mermaid
           class: mermaid
           format: !!python/name:pymdownx.superfences.fence_code_format
+        - name: python
+          class: python
+          validator: !!python/name:markdown_exec.validator
+          format: !!python/name:markdown_exec.formatter
```
Comment (pawamoy) [inline mkdocs.yml:90]: You don't need to declare the fence if you added `markdown-exec` to the plugin list. The plugin basically declares a fence for each of its supported language.

**Review by pawamoy — COMMENTED** [review]
Very nice! I'll probably use your findings in my own docs!

File: docs/stories/from-sphinx-to-mkdocs.md (hunk 1 above)
Comment (ssbarnea) [reply docs/stories/from-sphinx-to-mkdocs.md:116]: I will check to be sure. TBH, I find both confusing as I would have expected either `lang` or `formatter` as a better name.

In fact I would have wished the "ansi" was not even needed and just implied as I see zero use for displaying an ANSI stream as plain text. Still that bit is likely not in your extension control.

I will include the CSS in a plugin once I learn how to do it, to make it much easier to reuse.

**Pull request merged**